app/addons/documents/tests/resourcesSpec.js

```diff
@@ -460,5 +461,33 @@ describe('Bulk Delete', function () {
 
   });
 
+});
+
+// this should stay at the bottom due to manipulating the 'view' urls.  otherwise, update this to restore them.
+describe('IndexCollection', function () {
+  var collection;
+
+  beforeEach(function () {
+    FauxtonAPI.registerUrls('view', {
+      testWithTrailingQuestion: function (database, designDoc, viewName) {
+        return app.host + '/' + database + '/_design/' + designDoc + '/_view/' + viewName + "?bogusParam=foo";
+      }
+    });
+    collection = new Models.IndexCollection([{
+      id:'myId1',
+      doc: 'num1'
+    },
+    {
+      id:'myId2',
+      doc: 'num2'
+    }], {
+      database: {id: 'databaseId', safeID: function () { return this.id; }},
+      design: '_design/myDoc'
+    });
+  });
+
+  it('creates the right url with correct params when trailing question mark', function () {
```

Comment:
Can you add another test to prove it works when you don't add the trailing `?`. 
Also we don't use `function ()` anymore with tests. You can use arrow functions http://exploringjs.com/es6/ch_arrow-functions.html